exporter/metrics.go

```diff
@@ -353,64 +355,87 @@ func metricRenameAndLabel(rm *rawMetric, convs []conversion) *rawMetric {
 	return nil
 }
 
-// specialConversions returns a list of special conversions we want to implement.
-// See: https://jira.percona.com/browse/PMM-6506
-func specialConversions() []conversion {
-	return []conversion{
-		{
-			oldName:     "mongodb_ss_opLatencies_ops",
-			prefix:      "mongodb_ss_opLatencies",
-			suffixLabel: "op_type",
-			suffixMapping: map[string]string{
-				"commands_ops":     "commands",
-				"reads_ops":        "reads",
-				"transactions_ops": "transactions",
-				"writes_ops":       "writes",
-			},
+func specialMetricRenameAndLabel(rm *rawMetric) *rawMetric {
```

Comment:
what's the difference of this method from MetricRename and Label?

Reply:
They use a different set of rules. 
Since now rules are global to avoid passing/copying maps and the garbage collector work, I had to split into 2 different functions.